pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -665,7 +665,7 @@ public ServerReloadControllerJobStatusResponse getReloadJobStatus(
         serverToSegments.put(server, segmentList);
       });
     } else {
-      serverToSegments = _pinotHelixResourceManager.getServerToSegmentsMap(tableNameWithType);
+      serverToSegments = _pinotHelixResourceManager.getServerToOnlineSegmentsMap(tableNameWithType);
```

Comment:
i was actually thinking whether we should do 4 counts
1. zk segment_cnt
2. zk online_segment_cnt
3. SUM(segment_cnt) from each server 
4. SUM(success_cnt) from each server

the logic to return success or failure will be based off 3==4, but if 3 != 2 we should do a warning. thoughts @Jackie-Jiang